pkg/blobstore/cloud/cloud_blob_access_test.go

```diff
@@ -0,0 +1,108 @@
+package cloud_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/buildbarn/bb-storage/pkg/blobstore"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/cloud"
+	"github.com/buildbarn/bb-storage/pkg/digest"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"gocloud.dev/blob/memblob"
+)
+
+// These tests use memblob as a fake bucket. blob.Bucket isn't
+// reasonably mockable, its methods return struct types which can't be
```

Comment:
This is interesting, isn't it? gocloud is supposed to offer abstractions. Meanwhile blob.Bucket is a concretion that can't easily be mocked.

Reply:
It would be reasonable if memblob was a little more complete - it isn't quite rich enough to be a full fake, there's no way to test the error paths. I might send them a PR later.